lib/src/components/NxTooltip/NxTooltip.tsx

```diff
@@ -36,7 +37,8 @@ function fixOptional(props: Props): TooltipProps {
     onClose: props.onClose || undefined,
     open: props.open === null ? undefined : props.open,
     placement: convertPlacement(props.placement),
-    title: props.title || ''
+    title: props.title || '',
+    'aria-label': textContent(props.title)
```

Comment:
Btw this makes it read out twice in ChromeVox

Reply:
I wasn't able to duplicate this with ChromeVox locally.